packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -0,0 +1,267 @@
+import { zodResolver } from "@hookform/resolvers/zod";
```

Comment:
Abstracted away the form in its own component as well. This way we can test it separately instead of as part of the bigger page. We could decide in the future to make this fully reusable, but for now it relies on the booker's store to get its data, instead of having to pass everything in as props.